content/rc/rc-quickstart.md

```diff
@@ -114,7 +79,7 @@ These are displayed in the **Configuration** tab.
 
 Once you have the connection details, you can connect in a variety of ways, including:
 
-- Using the `redis-cli` utility 
+- Using the `redis-cli` utility
```

Comment:
what commands do they need to use with the redis cli? Usually we don't duplicate but in a quick start you want all the info they need at their fingertips.